ansibullbot/decorators/github.py

```diff
@@ -187,9 +187,12 @@ def inner(*args, **kwargs):
                 elif isinstance(e, httplib.BadStatusLine):
                     # https://github.com/ansible/ansibullbot/issues/602
                     stime = 2*60
-                elif hasattr(e, 'status') and e.status == 500:
+                elif getattr(e, 'status', None) == 500:
```

Comment:
Since you decided to touch this:
```suggestion
                elif getattr(e, 'status', None) > 500:
```
(or `in [500, 502]`)